src/components/ContactInfo/ContactInfo.vue

```diff
@@ -249,6 +249,10 @@ export default class ContactInfo extends Vue {
   @Prop()
   private editedLabel!: string
 
+  /** Option to disable the edit actions */
+  @Prop()
+  private disableActions!: boolean
```

Comment:
Since you don't have a default value for this prop (and you didn't make it required), should you set it in the story vue file?

Reply:
I should set a default for this one actually, that is important i think.

Reply:
Updated in next commit!